packages/iris-grid/src/IrisGridTableModelTemplate.ts

```diff
@@ -525,7 +525,16 @@ class IrisGridTableModelTemplate<
         return '*';
       }
     }
-    return text ?? '';
+
+    if (text === null) {
```

Comment:
Guessing this means exporting behaviour was consistent after this change? Excellent.